src/components/ui/Checkbox.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
```

Comment:
can we use the existing checkbox component pls? No need to reinvent the wheel. It's under Form/
There is no need to style it

Reply:
i worked with old version where i couldn't find any components i need

Reply:
Checkbox and Tab were available 2 weeks ago though?